frontend/src/api/todoApi.ts

```diff
@@ -1,8 +1,20 @@
 import axios from "axios";
-import { IUser } from "../models/ITodo";
+import { IMongoTodo, IUser } from "../models/ITodo";
 
 const fetchTodos = () => {
-  return axios.get(`https://mern-todo-app-a66w.onrender.com/`);
+  return axios.get(`https://mern-todo-app-a66w.onrender.com/todo`);
```

Comment:
You should have BASE_URL somewhere in config file or something like that to avoid duplications